back/engines/free/seo/spec/acceptance/application_spec.rb

```diff
@@ -72,12 +75,15 @@
 
       example 'the sitemap has the right number of items' do
         expected_count = base_count +
-                         (idea_count    * url_count_per_idea    * locales_count) +
-                         (project_count * url_count_per_project * locales_count) +
+                         (idea_count * url_count_per_idea * locales_count) +
+                         ((project_count + 2) * url_count_per_project * locales_count) +
                          (1 * locales_count) + # project folders
                          (1 * locales_count) + # initiatives
-                         (1 * locales_count) # static pages
```

Comment:
I think it's correct, for now I left initiatives in there deliberately (in general, only removed it if I had to), and static pages is still there, the diff just pops up because now the line ends with a `+`